dipy/direction/tests/test_bingham.py

```diff
@@ -0,0 +1,204 @@
+import numpy as np
+from numpy.testing import (assert_array_almost_equal, assert_almost_equal,
+                           assert_array_less)
+from dipy.direction.bingham import (_single_bingham_to_sf, _single_sf_to_bingham,
+                                    _bingham_fit_peak, bingham_fiber_density,
+                                    bingham_fiber_spread, sf_to_bingham,
+                                    _convert_bingham_pars, odi2k, k2odi,
+                                    sh_to_bingham)
+from dipy.data import get_sphere
+from dipy.reconst.shm import sf_to_sh
+
+sphere = get_sphere('repulsion724')
+sphere = sphere.subdivide(2)
+
+
+def test_bingham_fit():
+    """ Tests for bingham function and single Bingham fit"""
+    peak_dir = np.array([1, 0, 0])
+    ma_axis = np.array([0, 1, 0])
+    mi_axis = np.array([0, 0, 1])
+    k1 = 2
+    k2 = 6
+    f0 = 3
+
+    # Test if maximum amplitude is in the expected Bingham main direction
+    # which should be perpendicular to both ma_axis and mi_axis
+    odf_test = _single_bingham_to_sf(f0, k1, k2, ma_axis, mi_axis, peak_dir)
+    assert_almost_equal(odf_test, f0)
+
+    # Test Bingham fit on full sampled GT Bingham function
+    odf_gt = _single_bingham_to_sf(f0, k1, k2, ma_axis, mi_axis, sphere.vertices)
+    a0, c1, c2, mu0, mu1, mu2 = _bingham_fit_peak(odf_gt, peak_dir, sphere, 45)
```

Comment:
@RafaelNH If I replace `peak_dir` by `-peak_dir` I can lower the max angle to lesser than 45 and the test comparing `Mus` and Mus_ref` will still pass.